processor/src/deadwood_segmentation/unet_model.py

```
@@ -0,0 +1,49 @@
+
+from .unet_parts import *
+
+
+class UNet(nn.Module):
+	def __init__(self, n_channels, n_classes, bilinear=False):
+		super(UNet, self).__init__()
+		self.n_channels = n_channels
+		self.n_classes = n_classes
+		self.bilinear = bilinear
+
+		self.inc = DoubleConv(n_channels, 64)
```

Ruff (0.8.0)

`DoubleConv` may be undefined, or defined from star imports

(F405)